ui.js

```diff
@@ -1,49 +1,29 @@
 /*!
  * UI Utility module.
  *
- * Copyright (c) 2012-2016 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2012-2017 Digital Bazaar, Inc. All rights reserved.
  *
  * @author Dave Longley
  */
-define([
-  'angular',
-  './action-menu-directive',
-  './error-directive',
-  './headline-component',
-  './placeholder-directive',
-  './slug-filter',
-  './slug-in-directive',
-  './tabs-directive',
-  './tooltip-directive'
-], function(
-  angular,
-  actionMenuDirective,
-  errorDirective,
-  headlineComponent,
-  placeholderDirective,
-  slugFilter,
-  slugInDirective,
-  tabsDirective,
-  tooltipDirective) {
-
-'use strict';
+import angular from 'angular';
+import ActionMenuDirective from './action-menu-directive';
+import ErrorDirective from './error-directive';
+import HeadlineComponent from './headline-component';
+import PlaceholderDirective from './placeholder-directive';
+import SlugFilter from './slug-filter';
+import SlugInDirective from './slug-in-directive';
+import TabsComponent from './tabs-component';
+import TabsPaneDirective from './tabs-pane-directive';
+import TooltipDirective from './tooltip-directive';
 
 var module = angular.module('bedrock.ui', []);
 
-// TODO: convert other deps into `register` format and use a loop
-/*Array.prototype.slice.call(arguments, 1).forEach(function(dep) {
-  dep(module);
-});*/
-headlineComponent(module);
-
-module.directive(actionMenuDirective);
-module.directive(errorDirective);
-module.directive(placeholderDirective);
-module.filter(slugFilter);
-module.directive(slugInDirective);
-module.directive(tabsDirective);
-module.directive(tooltipDirective);
-
-return module.name;
-
-});
+module.component('brHeadline', HeadlineComponent);
+module.component('brTabs', TabsComponent);
+module.directive('brActionMenu', ActionMenuDirective);
+module.directive('brError', ErrorDirective);
+module.directive('brPlaceholder', PlaceholderDirective);
```

Comment:
Should just be `placeholder`.  Move the related comment from end of placeholder-directive.js here.